test/runnable/testconst.d

```diff
@@ -2806,9 +2806,9 @@ void test6982()
 /************************************/
 // 7038
 
```

Comment:
Since the code to implement this is generic for all qualifiers, that is unnecessary. But I'll add one for shared.